integration_tests/models/sql/test_star_quotes.sql

```diff
@@ -0,0 +1,10 @@
+with data as (
+
+    select
+        {{ dbt_utils.star(from=ref('data_star_quotes')) }}
+
+    from {{ ref('data_star_quotes') }}
+
+)
+
+select * from data
```

Comment:
Will the test fail if `quote_identifiers=False` is specified ? I _suspect_ not here either!

Reply:
It still succeeds, as you suspected! Removing this test altogether (will test inside of your suggested path).